google/cloud/spanner/internal/database_admin_logging_test.cc

```diff
@@ -57,7 +58,7 @@ TEST_F(DatabaseAdminLoggingTest, CreateDatabase) {
   DatabaseAdminLogging stub(mock_, TracingOptions{});
 
   CompletionQueue cq;
-  std::unique_ptr<grpc::ClientContext> context(new grpc::ClientContext);
+  auto context = absl::make_unique<grpc::ClientContext>();
```

Comment:
Completely optional, but doing

```cc
auto response = stub.AsyncCreateDatabase(cq, absl::make_unique<grpc::ClientContext>(), ...);
```

seems cleaner than:

```cc
auto context = absl::make_unique<grpc::ClientContext>();
auto response = stub.AsyncCreateDatabase(cq, std::move(context), ...);
```

Reply:
Yes, I hesitated on that originally, but done.  Thanks.